### ISSUE TITLE
StreetHazards color map

### ISSUE THREAD

**Issue by Mickey-Boy** [matejgrcic/DenseHybrid#8]
Thanks for your interesting work. I have encountered some visualization problems for StreetHazards dataset. I followed this issue to visualize the label map to RGB space: [color map](https://github.com/hendrycks/anomaly-seg/issues/15) but obtained the different images from your visualization shown in the paper. 

![rgb_1025](https://github.com/matejgrcic/DenseHybrid/assets/101962863/7d8eb399-3dbf-4109-9fa5-abc7fd29f676)

May you show your visualization codes for [evaluate_osr_sh.py](https://github.com/matejgrcic/DenseHybrid/blob/main/evaluate_osr_sh.py) for better comparison. Thanks.

**Comment by matejgrcic** [comment]
Hi, our visualization codes are already available here: https://github.com/matejgrcic/DenseHybrid/blob/main/data/datasets/street_hazards/streethazards_labels.py
These color maps are likely taken from an earlier version of the Streethazards dataset repo.

Also, note that we ignored class "other" in our training and evaluation setups since it lacks semantical meaning and we're dealing with open-set segmentation. This decision was also motivated by the fact that "other" objects (e.g. trash cans) were also ignored in Cityscapes taxonomy.

**Comment by Mickey-Boy** [comment]
Thanks for your response. Still can't figure out the visualization problem. May you share the code about how to generate DenseHybrid recognition results in evaluate_osr file. 

![image](https://github.com/matejgrcic/DenseHybrid/assets/101962863/84398cf4-669b-4d8d-aa4f-16449eccae0e)
For the overall open-set miou, I tested with weights (LDN-121 fine-tuned on StreetHazards with ADE20k negatives) but got a result different from that reported in the paper.
